lib/rollbax.ex

```diff
@@ -147,6 +147,34 @@ defmodule Rollbax do
     })
   end
 
+  @doc """
+  Reports the given `message`.
+
+  `message` will be reported as a simple Rollbar message, for example, without a stacktrace.
+  `level` is the level of the message, which can be one of:
+
+    * `:critical`
+    * `:error`
+    * `:warning`
+    * `:info`
+    * `:debug`
+
+  `custom` and `occurrence_data` work exactly like they do in `report/5`.
+
+  ## Examples
+
+      Rollbax.report_message(:critical, "Everything is on fire!")
+      #=> :ok
+
+  """
+  @spec report_message(:critical | :error | :warning | :info | :debug, IO.chardata, map, map) :: :ok
+  allowed_levels = [:critical, :error, :warning, :info, :debug]
```

Comment:
Attribute instead of unquoting?

Reply:
We won't use it elsewhere but if you prefer, sure, I have no personal preference 👍